server/src/termdb.matrix.js

```diff
@@ -153,12 +153,14 @@ async function getSampleData(q) {
 			if (tw.q?.mode == 'discrete') {
 				const min = tw.term.bins.min
 				const max = tw.term.bins.max
-				lst =
-					tw.q.type == 'regular-bin'
-						? compute_bins(tw.q, () => {
-								return { min, max }
-						  })
-						: tw.q.lst
+				if (tw.q.type == 'regular-bin') {
+					lst = compute_bins(tw.q, () => {
+						return { min, max }
+					})
+					for (const b of lst) {
+						if (!('name' in b) && b.label) b.name = b.label
```

Comment:
what's purpose of doing this?

Reply:
Adding to compute_bins will break many binning tests. In **_termdb.sql.numeric.js_**, name was added the same as above in getCTE. Maybe we could update compute_bins in another PR?

Reply:
sure let's fix it in new pr